src/components/Dashboard/PendingList/filings/ContinuationIn.vue

```diff
@@ -8,32 +8,12 @@
     <template #body>
       <div class="body-2">
         <v-divider class="mt-1 mb-4" />
-        <h4 v-if="isFutureEffectivePast || isFutureEffective">
```

Comment:
Please remove these getters (computed) also.

Reply:
And "effectiveDateTime".

Reply:
Removed